core/src/main/scala/pl/edu/agh/iet/akka_debugging/database/DatabaseUtils.scala

```diff
@@ -30,20 +34,17 @@ object DatabaseUtils {
     override def * = (id, sender, receiver) <> (CollectorDBMessage.tupled, CollectorDBMessage.unapply)
   }
 
-  class CollectorDBMessagesRelation(tag: Tag) extends Table[CollectorDBMessageRelation](tag, "relation") {
+  class CollectorDBMessagesRelations(tag: Tag) extends Table[CollectorDBMessagesRelation](tag, "relation") {
     def id1 = column[Int]("id1")
     def id2 = column[Int]("id2")
     def pk = primaryKey("pk", (id1, id2))
-    override def * = (id1, id2) <> (CollectorDBMessageRelation.tupled, CollectorDBMessageRelation.unapply)
+    override def * = (id1, id2) <> (CollectorDBMessagesRelation.tupled, CollectorDBMessagesRelation.unapply)
   }
 
-  def init(): Unit = {
-    import scala.concurrent.ExecutionContext.Implicits.global
-
+  def init(implicit ex: ExecutionContext): Future[Unit] = {
     val db = dc.db
```

Comment:
Is it kind of convention of shortening variables connected with database? (It's not very important - I simply like descriptive names ;) )